src/plots/gl3d/scene.js

```diff
@@ -839,15 +872,33 @@ proto.saveCamera = function saveCamera(layout) {
         }
     }
 
+    if(!hasChanged) {
+        if(aspectDataLastSave === undefined) {
+            hasChanged = true;
+        } else {
+            if(
+                aspectDataLastSave.x !== aspectData.x ||
+                aspectDataLastSave.y !== aspectData.y ||
+                aspectDataLastSave.z !== aspectData.z
+            ) {
+                hasChanged = true;
+            }
+        }
+    }
+
     if(hasChanged) {
         var preGUI = {};
         preGUI[this.id + '.camera'] = cameraDataLastSave;
+        preGUI[this.id + '.aspectratio'] = aspectDataLastSave;
```

Comment:
Here too, we should only store the aspectratio change for orthographic projections.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4292/commits/b92230a0b99f883dd009e89890afc7555450a7a2.